src/js/controllers/ctrl-scores.js

```diff
@@ -192,21 +191,22 @@ app.controller('scorePageController', function(Please, $scope, $q, $timeout, wid
 			// get the current attempt from the url
 			const hash = getAttemptNumberFromHash()
 			if (currentAttempt === hash) {
-				return
```

Comment:
this would prevent the promise from resolving, causing `_getScoreDetails()->then()` to not exist